operations/vouchers.md

```diff
@@ -361,7 +361,7 @@ Updates information about the specified vouchers. Note this operation supports [
 | `Name` | [String update value](_objects.md#string-update-value) | optional, max length 128 characters | Internal name of the voucher (or `null` if the name should not be updated). |
 | `Type` | [VoucherTypeUpdateValue](_objects.md#string-update-value) | optional | Type of the voucher e.g. 'Public', 'PartnerCompany' or 'TravelAgency' (or `null` if the type should not be updated). |
 | `CompanyId` | [String update value](_objects.md#string-update-value) | optional | Unique identifier of Company (Company or Travel Agency) the voucher is related to. This is required for Type of `PartnerCompany` or `TravelAgency`. Use `null` if Company should not be updated. |
-| `AssignedRateIds` | [GuidIEnumerableUpdateValue](_object.md#array-of-strings-update-value) | optional, max length 5 characters | Unique identifiers of Rates (or `null` should it not be updated). |
+| `AssignedRateIds` | [GuidIEnumerableUpdateValue](_object.md#array-of-strings-update-value) | optional, max length 10 characters | Unique identifiers of Rates (or `null` should it not be updated). |
```

Comment:
```suggestion
| `AssignedRateIds` | [GuidIEnumerableUpdateValue](_object.md#array-of-strings-update-value) | optional, max length 10 characters | Unique identifiers of Rates (or `null` should it not be updated). |
```
@jnv not sure how hard is to address it, so I leave it up to you. It should say items not characters 🤔

Reply:
This looks like an error in Swashbuckle / one of our filters. In OAS there is:

```
AssignedRateIds:
  title: GuidIEnumerableUpdateValue
  maxLength: 10
  allOf:
    - $ref: '#/components/schemas/GuidIEnumerableUpdateValue'
  description: Unique identifiers of Rates (or `null` should it not be updated).
  nullable: true
```

But `maxLength` is valid only on string types. I think it could be interaction of our filters + Swashbuckle's use of `allOf` and `MaxLength` interpretation which differs between string and enumerable.

Feel free to fix it manually here; why I could work around it in the generator, we need to address this in the monolith.